src/install.js

```diff
@@ -46,15 +46,46 @@ questions.optional = [
 	},
 ];
 
-function checkSetupFlag() {
-	let setupVal = install.values;
+function checkSetupFlagEnv() {
+	let setupVal = install.values || {};
+
+	const envConfMap = {
+		NODEBB_URL: 'url',
+		NODEBB_PORT: 'port',
+		NODEBB_ADMIN_USERNAME: 'admin:username',
+		NODEBB_ADMIN_PASSWORD: 'admin:password',
+		NODEBB_ADMIN_EMAIL: 'admin:email',
+		NODEBB_DB: 'database',
+		NODEBB_DB_HOST: 'host',
+		NODEBB_DB_PORT: 'port',
+		NODEBB_DB_USER: 'username',
+		NODEBB_DB_PASSWORD: 'password',
+		NODEBB_DB_NAME: 'database',
+		NODEBB_DB_SSL: 'ssl',
+	};
+
+	// Set setup values from env vars (if set)
+	winston.info('checking env vars for setup info...');
```

Comment:
Do you mind prepending `[install/checkSetupFlagEnv] ` to the `winston` calls in this method? Gives a bit more context when a namespace (of sorts) is part of an output message

Reply:
done